opal/mca/accelerator/rocm/accelerator_rocm.h

```diff
@@ -14,8 +14,17 @@
 #include "opal_config.h"
 
 #include <stdio.h>
+
+/* Not interested in warnings generated in hip_runtime_api.h */
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wundef"
+#pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
I noticed this comment for a similar behavior:
```
/* Clang won't quietly accept "-pedantic", but GCC versions older than ~4.8                                             
 * won't quietly accept "-Wpedanic".  The whole "#pragma GCC diagnostic ..."                                            
 * facility only was added to GCC as of version 4.6. */                                                                 
#    if defined(__clang__) || (defined(__GNUC__) && __GNUC__ >= 6)                                                      
#        pragma GCC diagnostic ignored "-Wpedantic"                                                                     
#    else                                                                                                               
#        pragma GCC diagnostic ignored "-pedantic"                                                                      
#    endif     
```
Do you want to account for that?

Reply:
yes, thank you, that is a good idea! I will push an update to include this in a bit

Reply:
@wckzhang I incorporated this change